first_file.py

```diff
@@ -0,0 +1,4 @@
+a = 2
```

Comment:
It might be useful to make a a bit higher, such that the sum is less easy to calculate by head.
```suggestion
a = 1208573
```